app/controllers/shelter_pets_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ShelterPetsController < ApplicationController
```

Comment:
YESSS!! Nice refactor to make a more RESTful controller set up for this legacy code!